component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -562,6 +580,7 @@ public static void redirectToEnableTOTPReqPage(HttpServletResponse response, Aut
      * @param skey     QR code claim
      * @throws AuthenticationFailedException On error while getting value for enrolUserInAuthenticationFlow
      */
+    @Deprecated
```

Comment:
Add a deprecation comment mentioning what is the new method that should be used

Reply:
fixed with fce8e8503344493ed0136f775e2bd5cd7761e345